### PR TITLE
Rewrite introduction for html-integrations

### PR DESCRIPTION
**Pull request by xaviripo**

### CODE REVIEW

**Review by dcanetma — CHANGES_REQUESTED** [review]
MathType logo on README's main headline is missing.
Would you mind to upload it?

**Comment by xaviripo** [comment]
@dcanetma My bad; done.

**Review by dcanetma — APPROVED** [review]
Ready to merge!

File: README.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
+**MathType Web** is the official open-source JavaScript library for [MathType by Wiris], the leading formula editor and equation writer for CMS, LMS, and web applications.
 
-This repository contains the plugins for the most popular HTML editors, such as [CKEditor], [Froala], and [TinyMCE].
-They are available for download on [npm](https://www.npmjs.com/~wiris).
-We also provide easy-to-use examples showing how to integrate MathType in [Angular] and [React] projects, or even your own custom HTML editor in your website.
-Our main goal is to enable web developers to integrate MathType in their web projects on the widest range of technologies available.
-We also try to keep the library light, simple, and free from third-party dependencies.
+This repository contains the plugins source-code for the most popular HTML editors, such as [CKEditor], [Froala], and [TinyMCE]. They are available from the [npmjs](https://www.npmjs.com/~wiris) directory. It also provides a collection of working examples, including step-by-step information, on how to integrate MathType in vanilla HTML, [Angular], [React] projects, or even your own custom web editor.
```
Comment (xaviripo) [inline README.md:7]: A couple ideas:

- Reorder the word "plugins" after "source code". On the first read I missed the word plugins and thought that you meant that we hosted code for CKEditor, etc.
- [npm branding](https://www.npmjs.com/policies/trademark) drops the "js".

```suggestion
This repository contains the source code of the plugins for the most popular HTML editors, such as [CKEditor], [Froala], and [TinyMCE]. They are available from the [npm](https://www.npmjs.com/~wiris) directory. It also provides a collection of working examples, including step-by-step information, on how to integrate MathType in vanilla HTML, [Angular], [React] projects, or even your own custom web editor.
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
 
-For usage conditions, check the [page](https://www.wiris.com/en/pricing/) on the Wiris website.
+Our main goal is to enable developers to integrate MathType web on the widest range of technologies available. We also try to keep the library light, simple, and free from third-party dependencies. Everything you need to build and deploy is in the sources.
+
+MathType can be used for free up to a certain level of uses per natural year,  [more details](https://www.wiris.com/en/pricing/).
```
Comment (xaviripo) [inline README.md:11]:
```suggestion
 MathType can be used for free up to a certain level of uses per natural year, [more details](https://www.wiris.com/en/pricing/).
```

**Review by carlawiris — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dcanetma) [reply README.md:7]: Accepted

**Pull request merged**